ops/services/alerts/app_service_metrics/_var.tf

```diff
@@ -67,9 +67,9 @@ variable "http_response_time_aggregation" {
   default = "Average"
 }
 
-variable "failed_http_2xx_threshold" {
-  // The resource that uses this value doesn't have a >= check, so we need n - 1 here
-  default = 24
+variable "http_2xx_failure_rate_threshold" {
+  # allows 34 failures out of 100,000 requests
+  default = 0.0034
```

Comment:
What's our current failure rate for 2xx errors? This seems lower than our current rate of 24 failures per total traffic.

Reply:
It looks like this decimal value is also causing the `terraform plan` to fail its run. This calculation may need a bit of refactoring to make the math work out correctly.

Reply:
Thanks for the callouts, I've been reworking a couple of things in this PR and will push updates soon. Feel like I'm getting a handle on this query language, too!